File: Core java/Day02Assignment/src/com/sonata/TShirt.java
package com.sonata;

public class TShirt {
	String color;
	String material;
	String design;
	public TShirt(String c, String m, String d) {
		this.color=c;
		this.design=d;
		this.material=m;
	}
	
	public void display() {
		System.out.println("Product color is:" + color);
		System.out.println("Product material is :" + material);
		System.out.println("Product design is:" + design);
		
	}
	public static void main(String[] args) {
		TShirt small = new TShirt("black", "jeans", "fit") ;
		TShirt large = new TShirt("blue", "cotton", "jauguar") ;
		TShirt xl = new TShirt("White", "silk", "barma") ;
		System.out.println("\nSmall Size Details:");
		small.display();
		
		System.out.println("\nlarge size Details:");
		large.display();
		
		System.out.println("\nxl Size Details:");
		xl.display();
	}

}

File: Typescript assignment and practiced problems/assignment/TShirt.ts
class Tshirt{
    color:string;
    material:string;
    design:string;
    size:string;

    constructor(siz:string,col:string,  matr:string,des:string){
        this.color = col;
        this.design = des;
        this.material = matr;
        this.size = siz;
    }
    display(){
        console.log("Size - " + this.size 
                + "\n Material - " + this.material + 
                    "\n Design - "  + this.design  
                    +"\n Color - " + this.color);
    }
}

var t1 = new Tshirt("small","red","cotton","round-neck");
var t2 = new Tshirt("large","yellow","lenin","V-neck");
var t3 = new Tshirt("xtra-large","blue","cotton","round-neck");

t1.display();
t2.display();
t3.display();